crates/router/src/connector/stripe/transformers.rs

```diff
@@ -2446,7 +2446,7 @@ impl<F> TryFrom<&types::RefundsRouterData<F>> for RefundRequest {
         Ok(Self {
             amount: Some(amount),
             payment_intent,
-            metadata_order_id: item.payment_id.clone(),
```

Comment:
And let's have a type for the metadata in the payment, refund web hooks and wherever metadata is necessary. So that any one who is making changes will be able to know that their changes will also affect other flows like Sync and web hooks.